homework/kaydunov_denis/shopping_list_application_part4/src/main/java/github/kaydunovdenis/bean/Product.java

```diff
@@ -0,0 +1,120 @@
+package github.kaydunovdenis.bean;
+
+import github.kaydunovdenis.service.ProductPrintService;
+
+import java.math.BigDecimal;
+
+public class Product {
+    private Long id;
+    private String name;
+    private BigDecimal price;
+    private ProductCategory productCategory;
+    private BigDecimal discount;
+    private String description;
+
+    public Product(Long id, String name, BigDecimal price, ProductCategory productCategory, BigDecimal discount,
+                   String description) {
+        this.id = id;
+        this.name = name;
+        this.price = price;
+        this.productCategory = productCategory;
+        this.discount = discount;
+        this.description = description;
```

Comment:
In general, 7 params is a conventional limit. Add Builder instead of complex constructor.

Reply:
Fixed. But I'm not sure if this is spelled correctly.